client/src/components/QuestionsAndAnswers/components/SearchBar.jsx

```diff
@@ -3,23 +3,24 @@ import "./QuestionAndAnswers.css";
 
 
 
+const SearchBar = ({ handleSearch }) => {
 
+  const [searchTerm, setSearchTerm] = useState('');
 
-const SearchBar = ({ handleSearch, searchTerm }) => {
 
   const onChange = (e) => {
+    setSearchTerm(e.target.value);
     handleSearch(e.target.value);
   }
 
   return (
 
-      <div className = 'searchBar'>
-        <input size = '100' type='text' placeholder='HAVE A QUESTION? SEARCH FOR ANSWERS...'
-        onChange = {onChange} value = {searchTerm} /><i className="fa fa-search"></i>
-
+      <div className = 'searchBarWrapper'>
+         <i className="fa fa-search"></i>
+        <input className = 'search' size = '100' type='text' placeholder='HAVE A QUESTION? SEARCH FOR ANSWERS...'
```

Comment:
@HuijunLu Let's use class names more specific to the widget to avoid conflicts. `(className = 'search'`)